content/docs/start/data-versioning.md

```diff
@@ -77,7 +77,7 @@ outs:
 
 > \* See
 > [Large Dataset Optimization](/doc/user-guide/large-dataset-optimization) and
-> `dvc config cache` for more information on file linking.
+> `dvc config cache` for more info on file linking.
```

Comment:
intentional?

Reply:
Yes, keeps the note in a single line when rendered. Could be "info." to make it a little more formal.

Reply:
BTW I copied this existing note to the use case (and changed to "info"), that's why I also edited it here — consistency.